app/controllers/api/guidance_controller.rb

```diff
@@ -0,0 +1,22 @@
+module API
+  class GuidanceController < ApplicationController
+    skip_before_action :authenticate
+
+    layout 'api_guidance', only: 'show'
+
+    def show
+    end
+
+    def page
+      path = params[:page]
+
+      template = {
+        "page-1" => "page_1",
+        "page-2" => "page_2",
+        "page-3" => "page_3",
+      }.fetch(path)
```

Comment:
I hardcoded these for now. When we have a better idea of which pages we'll have this can be moved to a service object that gives us a list of paths/templates/page titles etc we can use here and above in `#navigation_items`.

Reply:
I prefer service object to model (which we use in NPQ)